x-pack/legacy/plugins/infra/public/containers/logs/log_entries/index.ts

```diff
@@ -124,48 +138,92 @@ const useFetchEntriesEffect = (
   dispatch: Dispatch,
   props: LogEntriesProps
 ) => {
-  const { getLogEntriesAround, getLogEntriesBefore, getLogEntriesAfter } = useGraphQLQueries();
-
-  const [prevParams, cachePrevParams] = useState(props);
+  const [prevParams, cachePrevParams] = useState<LogEntriesProps | undefined>();
```

Comment:
We initialize `prevParams` without initial value now. We treat this case as "first load" and force a fetch